docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -18307,7 +18307,16 @@ array of 2 or fewer elements.
 
 | WordRegex
 | `(?-mix:\A(?:\p{Word}\|\p{Word}-\p{Word}\|\n\|\t)+\z)`
-| 
```

Comment:
The adoc files are automatically generated from the configuration and comments in the class, could you revert this file?